src/Propel/Runtime/ActiveQuery/ModelCriteria.php

```diff
@@ -1044,17 +1052,32 @@ public function addSelectQuery(Criteria $subQueryCriteria, $alias = null, $addAl
      *
      * @param bool $force To enforce adding columns for changed alias, set it to true (f.e. with sub selects)
      *
-     * @return $this The current object, for fluid interface
+     * @return $this|\Propel\Runtime\ActiveQuery\ModelCriteria
```

Comment:
I'm usually pretty wary of this PHPstorm-ism. This type-hint (reads like 'this OR a `ModelCriteria` object, meaning 'this function might return an object reference to a `ModelCriteria` _other than_ `$this`) is less specific than `$this`. With the [pure-intersection-types RFC](https://wiki.php.net/rfc/pure-intersection-types) around the corner for PHP8.1 we should probably use either `$this` (even though it's nonstandard PHPdoc) or `\Propel\Runtime\ActiveQuery\ModelCriteria&$this` (to help phpdoc attribute interpreters that handle neither intersection types and the special value `$this`

Wow, that was a long piece of text for such a small issue.
TL;DR: don't assume PHPstorm is the only editor in the world 😜

Reply:
Oh, I'm not using PhpStorm. If I remember correctly, one of stan or psalm complained about this somewhere, so I switched to always set both. It seems to work if I remove it here

Reply:
The error is back:
```
1057 | ERROR | Chainable method (@return $this) cannot have multiple
      |       | return types in code.
      |       | (Spryker.Commenting.DocBlockReturnSelf.InvalidChainable)
```
I'll change it back